src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -86,6 +86,7 @@ struct ComputeDuDt {
 template <size_t Dim>
 struct ComputeNormalDotFluxes {
   using argument_tags = tmpl::list<Pi, Phi<Dim>>;
+  using target_fields = tmpl::list<Pi, Phi<Dim>, Psi>;
```

Comment:
Why is this better than supplying the `package_tags`?

Reply:
I don't understand this question.  There is nothing called `package_tags` here and there is no operation going on that I would consider packaging.

Reply:
```
  using package_tags =
      tmpl::list<Tags::NormalDotFlux<Pi>, Tags::NormalDotFlux<Phi<Dim>>, Pi,
                 NormalTimesFluxPi>;
```
line 133 of the Equations.hpp in develop

Reply:
I agree the changes in this PR are not so obvious, it's probably better to talk about it in a hangout. We discussed this in a hangout yesterday (@wthrowe, @kidder, @osheamonn and I) and I'm happy to talk to you too when you have time. I'm also going to add some documentation to provide a better basis for discussion than the raw code.

For now:

- `target_fields` specifies the databox tags that a numerical flux computer applies to. That means its `()` operator is expected to compute `db::wrap_tags_in<Tags::NormalDotNumericalFlux, target_fields>`.
- To do that, the `()` operator needs the `package_tags` on both sides of the interface.
- The packaging, in turn, needs the `slice_tags` (to be renamed to `argument_tags`) on the interface, plus the unit normal.

The flux communication makes the packaged data available on both sides of the interface. Then in the action where the fluxes are applied, the `()` operator is called. Usually, the computed numerical fluxes F^star are then passed into `lift_flux`, and this is where also the N.F are needed since it computes the strong form -N.(F^star - F). We know which N.F we need here, namely `db::wrap_tags_in<Tags::NormalDotFlux, target_fields>`, so these are extracted from the databox.